platform.js

```diff
@@ -772,7 +772,8 @@
             name = 'Node.js';
             arch = data.arch;
             os = data.platform;
-            version = /[\d.]+/.exec(data.version)[0];
+            version = /[\d.]+/.exec(data.version)
+            version = (version && version.length > 0) ? version[0] : 'unknown';
```

Comment:
The check can be simplified. When performing an `exec` it will either be an array with at least 1 element or `null` so the check can be `version ? version[0] : 'unknown'`.